rules/windows/defense_evasion_volume_shadow_copy_deletion_via_powershell.toml

```
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2021/07/19"
+maturity = "production"
+updated_date = "2021/07/19"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies use of WMI’s Win32_ShadowCopy class or CIM cmdlets for shadow copy deletion on endpoints. This commonly occurs in tandem with ransomware or
```

I don't know if this makes a difference anywhere, but this is a switch to the ASCII `'`
```suggestion
Identifies use of WMI's Win32_ShadowCopy class or CIM cmdlets for shadow copy deletion on endpoints. This commonly occurs in tandem with ransomware or
```